app/views/comment_mailer/send_notification_email.html.erb

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to RubyGamedev, <%= @user.username %></h1>
```

Comment:
For the content of this email, please use the content in this design: https://invis.io/BX3SPZLWD